tee-worker/app-libs/stf/src/getter.rs

```diff
@@ -188,8 +195,15 @@ impl ExecuteGetter for TrustedGetterSigned {
 			// litentry
 			TrustedGetter::user_shielding_key(who) =>
 				IdentityManagement::user_shielding_keys(&who).map(|key| key.encode()),
-			TrustedGetter::id_graph_stats(_who) =>
-				IdentityManagement::id_graph_stats().map(|item| item.encode()),
+			TrustedGetter::id_graph(who) => Some(IdentityManagement::get_id_graph(&who).encode()),
+			TrustedGetter::challenge_code(who, identity) =>
+				IdentityManagement::challenge_codes(&who, &identity).map(|code| code.encode()),
+			// TODO: we need to re-think it
```

Comment:
You mean previous implementation returned whole id graph stats, not only slice of that `AccountId` ?